iio/iio_demo/iio_demo.c

```diff
@@ -53,6 +53,13 @@
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
 
+#ifdef ADUCM_PLATFORM
+
+#define MAX_BUFF_SIZE 5000
+static uint8_t g_buff[MAX_BUFF_SIZE];
```

Comment:
nitpick: this could also be a buffer passed from main. This would have the benefit that it keeps the initialization structure identical across platforms. Would it be worth to do it this way?

Reply:
moved to main